app/src/androidTest/java/com/example/evently/CreateEventFragmentTest.java

```diff
@@ -124,4 +131,34 @@ protected int getGraph() {
     protected Class<CreateEventFragment> getFragmentClass() {
         return CreateEventFragment.class;
     }
+
+    private void setEventSchedule(
+            LocalDate selectionDate, LocalDate eventDate, LocalTime eventTime) {
+        scenario.onFragment(fragment -> {
+            setPrivateField(fragment, "selectionDeadline", selectionDate);
+            setPrivateField(fragment, "eventDate", eventDate);
+            setPrivateField(fragment, "eventTime", eventTime);
```

Comment:
Is it possible to input the dates via espresso? I think you can set date pickers in espresso with:

```java
onView(withClassName(Matchers.equalTo(DatePicker.class.getName()))).perform(setDate(year, monthOfYear, dayOfMonth));
```

or something like that. I know the `.perform(setDate(...` part should be right but there might be a better way to identify the view.